Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/clock_day_pm_.png and HtcClockWidget.apk/res/drawable-ldpi/clock_day_pm_.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/clock_tab_down_shadow.png and HtcClockWidget.apk/res/drawable-ldpi/clock_tab_down_shadow.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/digital_am.png and HtcClockWidget.apk/res/drawable-ldpi/digital_am.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/digital_divider.png and HtcClockWidget.apk/res/drawable-ldpi/digital_divider.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/digital_panel.png and HtcClockWidget.apk/res/drawable-ldpi/digital_panel.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/digital_pm.png and HtcClockWidget.apk/res/drawable-ldpi/digital_pm.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/dot.png and HtcClockWidget.apk/res/drawable-ldpi/dot.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_0.png and HtcClockWidget.apk/res/drawable-ldpi/flip_0.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_1.png and HtcClockWidget.apk/res/drawable-ldpi/flip_1.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_2.png and HtcClockWidget.apk/res/drawable-ldpi/flip_2.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_3.png and HtcClockWidget.apk/res/drawable-ldpi/flip_3.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_4.png and HtcClockWidget.apk/res/drawable-ldpi/flip_4.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_5.png and HtcClockWidget.apk/res/drawable-ldpi/flip_5.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_6.png and HtcClockWidget.apk/res/drawable-ldpi/flip_6.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_7.png and HtcClockWidget.apk/res/drawable-ldpi/flip_7.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_8.png and HtcClockWidget.apk/res/drawable-ldpi/flip_8.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_9.png and HtcClockWidget.apk/res/drawable-ldpi/flip_9.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_am.png and HtcClockWidget.apk/res/drawable-ldpi/flip_am.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_clock_divider.png and HtcClockWidget.apk/res/drawable-ldpi/flip_clock_divider.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/flip_pm.png and HtcClockWidget.apk/res/drawable-ldpi/flip_pm.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/location_weather_icon.png and HtcClockWidget.apk/res/drawable-ldpi/location_weather_icon.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_0.png and HtcClockWidget.apk/res/drawable-ldpi/number_0.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_1.png and HtcClockWidget.apk/res/drawable-ldpi/number_1.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_2.png and HtcClockWidget.apk/res/drawable-ldpi/number_2.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_3.png and HtcClockWidget.apk/res/drawable-ldpi/number_3.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_4.png and HtcClockWidget.apk/res/drawable-ldpi/number_4.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_5.png and HtcClockWidget.apk/res/drawable-ldpi/number_5.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_6.png and HtcClockWidget.apk/res/drawable-ldpi/number_6.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_7.png and HtcClockWidget.apk/res/drawable-ldpi/number_7.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_8.png and HtcClockWidget.apk/res/drawable-ldpi/number_8.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/number_9.png and HtcClockWidget.apk/res/drawable-ldpi/number_9.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/travel_clock_center_dot.png and HtcClockWidget.apk/res/drawable-ldpi/travel_clock_center_dot.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/weather_clock_tab_down.png and HtcClockWidget.apk/res/drawable-ldpi/weather_clock_tab_down.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/weather_clock_tab.png and HtcClockWidget.apk/res/drawable-ldpi/weather_clock_tab.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-ldpi/weather_clock_tab_up.png and HtcClockWidget.apk/res/drawable-ldpi/weather_clock_tab_up.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/clock_day_pm_.png and HtcClockWidget.apk/res/drawable-mdpi/clock_day_pm_.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/clock_tab_down_shadow.png and HtcClockWidget.apk/res/drawable-mdpi/clock_tab_down_shadow.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/digital_am.png and HtcClockWidget.apk/res/drawable-mdpi/digital_am.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/digital_divider.png and HtcClockWidget.apk/res/drawable-mdpi/digital_divider.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/digital_panel.png and HtcClockWidget.apk/res/drawable-mdpi/digital_panel.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/digital_pm.png and HtcClockWidget.apk/res/drawable-mdpi/digital_pm.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/dot.png and HtcClockWidget.apk/res/drawable-mdpi/dot.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_0.png and HtcClockWidget.apk/res/drawable-mdpi/flip_0.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_1.png and HtcClockWidget.apk/res/drawable-mdpi/flip_1.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_2.png and HtcClockWidget.apk/res/drawable-mdpi/flip_2.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_3.png and HtcClockWidget.apk/res/drawable-mdpi/flip_3.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_4.png and HtcClockWidget.apk/res/drawable-mdpi/flip_4.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_5.png and HtcClockWidget.apk/res/drawable-mdpi/flip_5.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_6.png and HtcClockWidget.apk/res/drawable-mdpi/flip_6.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_7.png and HtcClockWidget.apk/res/drawable-mdpi/flip_7.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_8.png and HtcClockWidget.apk/res/drawable-mdpi/flip_8.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_9.png and HtcClockWidget.apk/res/drawable-mdpi/flip_9.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_am.png and HtcClockWidget.apk/res/drawable-mdpi/flip_am.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_clock_divider.png and HtcClockWidget.apk/res/drawable-mdpi/flip_clock_divider.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/flip_pm.png and HtcClockWidget.apk/res/drawable-mdpi/flip_pm.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/location_weather_icon.png and HtcClockWidget.apk/res/drawable-mdpi/location_weather_icon.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_0.png and HtcClockWidget.apk/res/drawable-mdpi/number_0.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_1.png and HtcClockWidget.apk/res/drawable-mdpi/number_1.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_2.png and HtcClockWidget.apk/res/drawable-mdpi/number_2.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_3.png and HtcClockWidget.apk/res/drawable-mdpi/number_3.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_4.png and HtcClockWidget.apk/res/drawable-mdpi/number_4.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_5.png and HtcClockWidget.apk/res/drawable-mdpi/number_5.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_6.png and HtcClockWidget.apk/res/drawable-mdpi/number_6.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_7.png and HtcClockWidget.apk/res/drawable-mdpi/number_7.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_8.png and HtcClockWidget.apk/res/drawable-mdpi/number_8.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/number_9.png and HtcClockWidget.apk/res/drawable-mdpi/number_9.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/travel_clock_center_dot.png and HtcClockWidget.apk/res/drawable-mdpi/travel_clock_center_dot.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/weather_clock_tab_down.png and HtcClockWidget.apk/res/drawable-mdpi/weather_clock_tab_down.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/weather_clock_tab.png and HtcClockWidget.apk/res/drawable-mdpi/weather_clock_tab.png differ
Binary files ../original-apks/HtcClockWidget.apk/res/drawable-mdpi/weather_clock_tab_up.png and HtcClockWidget.apk/res/drawable-mdpi/weather_clock_tab_up.png differ
diff -rupN ../original-apks/HtcClockWidget.apk/res/values-land-mdpi/dimens.xml HtcClockWidget.apk/res/values-land-mdpi/dimens.xml
--- ../original-apks/HtcClockWidget.apk/res/values-land-mdpi/dimens.xml	2012-05-26 10:16:10.000000000 +0100
+++ HtcClockWidget.apk/res/values-land-mdpi/dimens.xml	2012-05-26 10:07:48.000000000 +0100
@@ -1,77 +1,77 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="sun_margin_bottom">0.0px</dimen>
-    <dimen name="todayinfo_margin_left">250.0px</dimen>
-    <dimen name="todayinfo_margin_top">13.0px</dimen>
-    <dimen name="todaytempview_margin_left">250.0px</dimen>
-    <dimen name="todaytempview_margin_top">67.0px</dimen>
-    <dimen name="high_temp_margin_top">-6.0px</dimen>
-    <dimen name="weatherclock_base_width">394.0px</dimen>
-    <dimen name="weatherclock_base_height">136.0px</dimen>
-    <dimen name="weatherclock_base_margin_left">16.0px</dimen>
-    <dimen name="weatherclock_base_margin_top">16.0px</dimen>
-    <dimen name="weather_view_margin_left">2.0px</dimen>
-    <dimen name="weather_view_margin_bottom">2.0px</dimen>
-    <dimen name="weather_press_width">394.0px</dimen>
-    <dimen name="weather_press_height">136.0px</dimen>
-    <dimen name="weather_press_margin_left">16.0px</dimen>
-    <dimen name="weather_press_margin_top">15.0px</dimen>
-    <dimen name="sun_width">118.0px</dimen>
+    <dimen name="todayinfo_margin_left">189.0px</dimen>
+    <dimen name="todayinfo_margin_top">5.0px</dimen>
+    <dimen name="todaytempview_margin_left">189.0px</dimen>
+    <dimen name="todaytempview_margin_top">50.0px</dimen>
+    <dimen name="high_temp_margin_top">-8.0px</dimen>
+    <dimen name="weatherclock_base_width">328.0px</dimen>
+    <dimen name="weatherclock_base_height">96.0px</dimen>
+    <dimen name="weatherclock_base_margin_left">12.0px</dimen>
+    <dimen name="weatherclock_base_margin_top">12.0px</dimen>
+    <dimen name="weather_view_margin_left">0.0px</dimen>
+    <dimen name="weather_view_margin_bottom">5.0px</dimen>
+    <dimen name="weather_press_width">328.0px</dimen>
+    <dimen name="weather_press_height">96.0px</dimen>
+    <dimen name="weather_press_margin_left">28.0px</dimen>
+    <dimen name="weather_press_margin_top">12.0px</dimen>
+    <dimen name="sun_width">84.0px</dimen>
     <dimen name="sun_height">79.0px</dimen>
-    <dimen name="sun_margin_left">91.0px</dimen>
-    <dimen name="sun_margin_top">70.0px</dimen>
-    <dimen name="digitalclock_width">260.0px</dimen>
-    <dimen name="digitalclock_height">114.0px</dimen>
-    <dimen name="digitalclock_margin_top">-16.0px</dimen>
-    <dimen name="tab_width">103.0px</dimen>
-    <dimen name="tab_height">114.0px</dimen>
-    <dimen name="tab_margin_leftright">24.0px</dimen>
-    <dimen name="tab_down_height">47.0px</dimen>
+    <dimen name="sun_margin_left">72.0px</dimen>
+    <dimen name="sun_margin_top">43.0px</dimen>
+    <dimen name="digitalclock_width">196.0px</dimen>
+    <dimen name="digitalclock_height">77.0px</dimen>
+    <dimen name="digitalclock_margin_top">-11.0px</dimen>
+    <dimen name="tab_width">74.0px</dimen>
+    <dimen name="tab_height">77.0px</dimen>
+    <dimen name="tab_margin_leftright">20.0px</dimen>
+    <dimen name="tab_down_height">34.0px</dimen>
     <dimen name="tab_down_margin_top">0.0px</dimen>
-    <dimen name="tab_updown_margin_left">7.0px</dimen>
-    <dimen name="tab_up_margin_top">8.0px</dimen>
-    <dimen name="digital_width">45.0px</dimen>
-    <dimen name="digital_height">73.0px</dimen>
-    <dimen name="digital_up_height">36.0px</dimen>
-    <dimen name="digital_margin_top">13.0px</dimen>
-    <dimen name="digital_up_margin_top">13.0px</dimen>
-    <dimen name="digital_down_margin_top">-36.0px</dimen>
+    <dimen name="tab_updown_margin_left">4.0px</dimen>
+    <dimen name="tab_up_margin_top">5.0px</dimen>
+    <dimen name="digital_width">29.0px</dimen>
+    <dimen name="digital_height">50.0px</dimen>
+    <dimen name="digital_up_height">24.0px</dimen>
+    <dimen name="digital_margin_top">10.0px</dimen>
+    <dimen name="digital_up_margin_top">10.0px</dimen>
+    <dimen name="digital_down_margin_top">-24.0px</dimen>
     <dimen name="digital_ten_margin_left">9.0px</dimen>
-    <dimen name="digital_units_margin_left">53.0px</dimen>
-    <dimen name="digital_center_margin_left">29.0px</dimen>
-    <dimen name="container_down_width">103.0px</dimen>
-    <dimen name="container_down_height">47.0px</dimen>
-    <dimen name="container_down_margin_top">49.0px</dimen>
-    <dimen name="container_up_width">103.0px</dimen>
-    <dimen name="container_up_height">50.0px</dimen>
-    <dimen name="ampm_width">13.0px</dimen>
-    <dimen name="ampm_height">8.0px</dimen>
-    <dimen name="ampm_margin_left">12.0px</dimen>
-    <dimen name="ampm_margin_top">75.0px</dimen>
-    <dimen name="ampm_down_margin_top">26.0px</dimen>
-    <dimen name="mask_width">99.0px</dimen>
-    <dimen name="mask_height">47.0px</dimen>
+    <dimen name="digital_units_margin_left">37.0px</dimen>
+    <dimen name="digital_center_margin_left">23.0px</dimen>
+    <dimen name="container_down_width">74.0px</dimen>
+    <dimen name="container_down_height">34.0px</dimen>
+    <dimen name="container_down_margin_top">34.0px</dimen>
+    <dimen name="container_up_width">74.0px</dimen>
+    <dimen name="container_up_height">36.0px</dimen>
+    <dimen name="ampm_width">10.0px</dimen>
+    <dimen name="ampm_height">6.0px</dimen>
+    <dimen name="ampm_margin_left">10.0px</dimen>
+    <dimen name="ampm_margin_top">54.0px</dimen>
+    <dimen name="ampm_down_margin_top">20.0px</dimen>
+    <dimen name="mask_width">72.0px</dimen>
+    <dimen name="mask_height">32.0px</dimen>
     <dimen name="mask_margin_left">2.0px</dimen>
-    <dimen name="mask_margin_top">46.0px</dimen>
-    <dimen name="divider_width">89.0px</dimen>
+    <dimen name="mask_margin_top">32.0px</dimen>
+    <dimen name="divider_width">64.0px</dimen>
     <dimen name="divider_height">1.0px</dimen>
-    <dimen name="divider_margin_left">7.0px</dimen>
-    <dimen name="divider_margin_top">49.0px</dimen>
-    <dimen name="dualclock_citytext_size">13.0sp</dimen>
-    <dimen name="dualclock_citytext_padding_left">9.0px</dimen>
-    <dimen name="dualclock_citytext_padding_right">9.0px</dimen>
-    <dimen name="dualclock_citytext_margin_bottom">0.0px</dimen>
-    <dimen name="dualclock_base_width">394.0px</dimen>
-    <dimen name="dualclock_base_height">109.0px</dimen>
-    <dimen name="dualclock_base_margin_top">13.0px</dimen>
-    <dimen name="dualclock_clocklayout_width">215.0px</dimen>
-    <dimen name="dualclock_clocklayout_height">155.0px</dimen>
-    <dimen name="dualclock_clocklayout_margin_left">-6.0px</dimen>
-    <dimen name="dualclock_clock_width">126.0px</dimen>
-    <dimen name="dualclock_clock_height">125.0px</dimen>
-    <dimen name="dualclock_clock_margin_top">2.0px</dimen>
-    <dimen name="dualclock_label_marginTop">170.0px</dimen>
-    <dimen name="dualclock_label_textsize">12.0px</dimen>
+    <dimen name="divider_margin_left">5.0px</dimen>
+    <dimen name="divider_margin_top">34.0px</dimen>
+    <dimen name="dualclock_citytext_size">11.0px</dimen>
+    <dimen name="dualclock_citytext_padding_left">5.0px</dimen>
+    <dimen name="dualclock_citytext_padding_right">5.0px</dimen>
+    <dimen name="dualclock_citytext_margin_bottom">20.0px</dimen>
+    <dimen name="dualclock_base_width">300.0px</dimen>
+    <dimen name="dualclock_base_height">91.0px</dimen>
+    <dimen name="dualclock_base_margin_top">9.0px</dimen>
+    <dimen name="dualclock_clocklayout_width">150.0px</dimen>
+    <dimen name="dualclock_clocklayout_height">120.0px</dimen>
+    <dimen name="dualclock_clocklayout_margin_left">28.0px</dimen>
+    <dimen name="dualclock_clock_width">104.0px</dimen>
+    <dimen name="dualclock_clock_height">104.0px</dimen>
+    <dimen name="dualclock_clock_margin_top">4.0px</dimen>
+    <dimen name="dualclock_label_marginTop">90.0px</dimen>
+    <dimen name="dualclock_label_textsize">10.0px</dimen>
     <dimen name="dualclock_marginBottom">0.0px</dimen>
     <dimen name="analogclock_clocklayout_width">212.0px</dimen>
     <dimen name="analogclock_clocklayout_height">145.0px</dimen>
@@ -83,68 +83,68 @@
     <dimen name="analogclock_clock_width">125.0px</dimen>
     <dimen name="analogclock_clock_height">125.0px</dimen>
     <dimen name="analogclock_clock_margin_top">0.0px</dimen>
-    <dimen name="analogclock_clock_ac_radius">34.0px</dimen>
-    <dimen name="analogclock_clock_ac_text_size">9.0px</dimen>
-    <dimen name="analogclock_clock_ac_data_width">25.0px</dimen>
-    <dimen name="analogclock44_clocklayout_width">430.0px</dimen>
-    <dimen name="analogclock44_clocklayout_height">219.0px</dimen>
-    <dimen name="analogclock44_clocklayout_padding_top">2.0px</dimen>
-    <dimen name="analogclock44_citytext_size">13.0sp</dimen>
-    <dimen name="analogclock44_citytext_padding_left">9.0px</dimen>
-    <dimen name="analogclock44_citytext_padding_right">9.0px</dimen>
+    <dimen name="analogclock_clock_ac_radius">25.0px</dimen>
+    <dimen name="analogclock_clock_ac_text_size">7.0px</dimen>
+    <dimen name="analogclock_clock_ac_data_width">20.0px</dimen>
+    <dimen name="analogclock44_clocklayout_width">360.0px</dimen>
+    <dimen name="analogclock44_clocklayout_height">163.0px</dimen>
+    <dimen name="analogclock44_clocklayout_padding_top">1.0px</dimen>
+    <dimen name="analogclock44_citytext_size">12.0px</dimen>
+    <dimen name="analogclock44_citytext_padding_left">5.0px</dimen>
+    <dimen name="analogclock44_citytext_padding_right">5.0px</dimen>
     <dimen name="analogclock44_citytext_margin_bottom">0.0px</dimen>
-    <dimen name="analogclock44_clock_width">196.0px</dimen>
-    <dimen name="analogclock44_clock_height">196.0px</dimen>
+    <dimen name="analogclock44_clock_width">144.0px</dimen>
+    <dimen name="analogclock44_clock_height">143.0px</dimen>
     <dimen name="analogclock44_clock_margin_top">0.0px</dimen>
-    <dimen name="analogclock44_clock_ac_radius">48.0px</dimen>
-    <dimen name="analogclock44_clock_ac_text_size">13.0px</dimen>
-    <dimen name="analogclock44_clock_ac_data_width">36.0px</dimen>
-    <dimen name="analogclock44_hit_width">200.0px</dimen>
-    <dimen name="analogclock44_hit_height">220.0px</dimen>
+    <dimen name="analogclock44_clock_ac_radius">32.0px</dimen>
+    <dimen name="analogclock44_clock_ac_text_size">8.0px</dimen>
+    <dimen name="analogclock44_clock_ac_data_width">26.0px</dimen>
+    <dimen name="analogclock44_hit_width">180.0px</dimen>
+    <dimen name="analogclock44_hit_height">190.0px</dimen>
     <dimen name="analogclock44_hit_margin_top">0.0px</dimen>
-    <dimen name="analogclock44_hit2_width">200.0px</dimen>
-    <dimen name="analogclock44_hit2_height">200.0px</dimen>
+    <dimen name="analogclock44_hit2_width">0.0px</dimen>
+    <dimen name="analogclock44_hit2_height">0.0px</dimen>
     <dimen name="analogclock44_hit2_margin_top">0.0px</dimen>
-    <dimen name="analogclockadj_clocklayout_width">430.0px</dimen>
-    <dimen name="analogclockadj_clocklayout_height">219.0px</dimen>
-    <dimen name="analogclockadj_clocklayout_padding_top">1.0px</dimen>
-    <dimen name="analogclockadj_citytext_size">13.0sp</dimen>
-    <dimen name="analogclockadj_citytext_padding_left">9.0px</dimen>
-    <dimen name="analogclockadj_citytext_padding_right">9.0px</dimen>
+    <dimen name="analogclockadj_clocklayout_width">380.0px</dimen>
+    <dimen name="analogclockadj_clocklayout_height">163.0px</dimen>
+    <dimen name="analogclockadj_clocklayout_padding_top">0.0px</dimen>
+    <dimen name="analogclockadj_citytext_size">12.0px</dimen>
+    <dimen name="analogclockadj_citytext_padding_left">5.0px</dimen>
+    <dimen name="analogclockadj_citytext_padding_right">5.0px</dimen>
     <dimen name="analogclockadj_citytext_margin_bottom">0.0px</dimen>
-    <dimen name="analogclockadj_hit_width">200.0px</dimen>
-    <dimen name="analogclockadj_hit_height">200.0px</dimen>
+    <dimen name="analogclockadj_hit_width">150.0px</dimen>
+    <dimen name="analogclockadj_hit_height">250.0px</dimen>
     <dimen name="analogclockadj_hit_margin_top">0.0px</dimen>
-    <dimen name="analogclockadj_hit2_width">200.0px</dimen>
-    <dimen name="analogclockadj_hit2_height">200.0px</dimen>
-    <dimen name="analogclockadj_hit2_margin_top">0.0px</dimen>
-    <dimen name="digitalclock_layout_padding_top">0.0px</dimen>
-    <dimen name="digitalclock_main_width">182.0px</dimen>
-    <dimen name="digitalclock_main_height">70.0px</dimen>
-    <dimen name="digitalclock_digital_width">26.0px</dimen>
-    <dimen name="digitalclock_digital_height">34.0px</dimen>
-    <dimen name="digitalclock_hour_tens_margin_left">32.0px</dimen>
-    <dimen name="digitalclock_hour_tens_margin_top">2.0px</dimen>
+    <dimen name="analogclockadj_hit2_width">250.0px</dimen>
+    <dimen name="analogclockadj_hit2_height">150.0px</dimen>
+    <dimen name="analogclockadj_hit2_margin_top">50.0px</dimen>
+    <dimen name="digitalclock_layout_padding_top">2.0px</dimen>
+    <dimen name="digitalclock_main_width">152.0px</dimen>
+    <dimen name="digitalclock_main_height">51.0px</dimen>
+    <dimen name="digitalclock_digital_width">16.0px</dimen>
+    <dimen name="digitalclock_digital_height">21.0px</dimen>
+    <dimen name="digitalclock_hour_tens_margin_left">37.0px</dimen>
+    <dimen name="digitalclock_hour_tens_margin_top">3.0px</dimen>
     <dimen name="digitalclock_hour_units_margin_left">-1.0px</dimen>
     <dimen name="digitalclock_dot_width">0.0px</dimen>
     <dimen name="digitalclock_dot_height">0.0px</dimen>
     <dimen name="digitalclock_dot_margin_left">0.0px</dimen>
     <dimen name="digitalclock_dot_margin_top">0.0px</dimen>
-    <dimen name="digitalclock_minute_tens_margin_left">13.0px</dimen>
+    <dimen name="digitalclock_minute_tens_margin_left">11.0px</dimen>
     <dimen name="digitalclock_minute_units_margin_left">-1.0px</dimen>
     <dimen name="digitalclock_data_margin_left">10.0px</dimen>
     <dimen name="digitalclock_data_margin_top">0.0px</dimen>
-    <dimen name="digitalclock_data_text_size">11.0sp</dimen>
-    <dimen name="digitalclock_ampm_width">17.0px</dimen>
-    <dimen name="digitalclock_ampm_height">10.0px</dimen>
-    <dimen name="digitalclock_ampm_margin_right">10.0px</dimen>
-    <dimen name="digitalclock_ampm_margin_top">2.0px</dimen>
-    <dimen name="digitalclock_divider_width">180.0px</dimen>
-    <dimen name="digitalclock_divider_height">2.0px</dimen>
-    <dimen name="digitalclock_divider_margin_left">3.0px</dimen>
-    <dimen name="digitalclock_divider_margin_top">2.0px</dimen>
-    <dimen name="digitalclock_city_width">129.0px</dimen>
-    <dimen name="digitalclock_city_margin_top">-1.0px</dimen>
-    <dimen name="digitalclock_city_text_size">12.0sp</dimen>
+    <dimen name="digitalclock_data_text_size">12.0sp</dimen>
+    <dimen name="digitalclock_ampm_width">14.0px</dimen>
+    <dimen name="digitalclock_ampm_height">8.0px</dimen>
+    <dimen name="digitalclock_ampm_margin_right">8.0px</dimen>
+    <dimen name="digitalclock_ampm_margin_top">3.0px</dimen>
+    <dimen name="digitalclock_divider_width">152.0px</dimen>
+    <dimen name="digitalclock_divider_height">1.0px</dimen>
+    <dimen name="digitalclock_divider_margin_left">0.0px</dimen>
+    <dimen name="digitalclock_divider_margin_top">1.0px</dimen>
+    <dimen name="digitalclock_city_width">99.0px</dimen>
+    <dimen name="digitalclock_city_margin_top">-2.0px</dimen>
+    <dimen name="digitalclock_city_text_size">11.0px</dimen>
     <dimen name="todayview_height">166.0px</dimen>
 </resources>
diff -rupN ../original-apks/HtcClockWidget.apk/res/values-mdpi/dimens.xml HtcClockWidget.apk/res/values-mdpi/dimens.xml
--- ../original-apks/HtcClockWidget.apk/res/values-mdpi/dimens.xml	2012-05-26 10:16:10.000000000 +0100
+++ HtcClockWidget.apk/res/values-mdpi/dimens.xml	2012-05-26 10:07:48.000000000 +0100
@@ -1,181 +1,181 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="default_row_height">50.0px</dimen>
-    <dimen name="sun_margin_bottom">12.0px</dimen>
+    <dimen name="default_row_height">40.0px</dimen>
+    <dimen name="sun_margin_bottom">1.0px</dimen>
     <dimen name="social_msg_text_size">@com.htc:dimen/darklist_primary_m</dimen>
     <dimen name="social_time_text_size">@com.htc:dimen/darklist_secondary_s</dimen>
     <dimen name="social_comment_text_size">@com.htc:dimen/darklist_secondary_s</dimen>
     <dimen name="social_like_text_size">@com.htc:dimen/darklist_secondary_s</dimen>
-    <dimen name="citytext_padding_top">1.0px</dimen>
-    <dimen name="default_check_height">37.0px</dimen>
-    <dimen name="cityname_text_size">14.0sp</dimen>
-    <dimen name="todaytemp_text_size">12.0sp</dimen>
-    <dimen name="crossview_text_size">12.0sp</dimen>
-    <dimen name="temp_big_text_size">31.0sp</dimen>
-    <dimen name="weekday_text_size">12.0sp</dimen>
-    <dimen name="todayinfo_margin_left">8.0px</dimen>
-    <dimen name="todayinfo_margin_top">113.0px</dimen>
-    <dimen name="cityname_shadow_max_width">140.0px</dimen>
-    <dimen name="cityname_shadow_layout_height">19.0px</dimen>
-    <dimen name="cityname_margin_top">8.0px</dimen>
+    <dimen name="citytext_padding_top">-2.0px</dimen>
+    <dimen name="default_check_height">40.0px</dimen>
+    <dimen name="cityname_text_size">12.0px</dimen>
+    <dimen name="todaytemp_text_size">11.0px</dimen>
+    <dimen name="crossview_text_size">11.0px</dimen>
+    <dimen name="temp_big_text_size">23.0px</dimen>
+    <dimen name="weekday_text_size">11.0px</dimen>
+    <dimen name="todayinfo_margin_left">5.0px</dimen>
+    <dimen name="todayinfo_margin_top">78.0px</dimen>
+    <dimen name="cityname_shadow_max_width">110.0px</dimen>
+    <dimen name="cityname_shadow_layout_height">17.0px</dimen>
+    <dimen name="cityname_margin_top">0.0px</dimen>
     <dimen name="crossview_margin_top">-3.0px</dimen>
-    <dimen name="todaytempview_margin_left">150.0px</dimen>
-    <dimen name="todaytempview_margin_top">119.0px</dimen>
+    <dimen name="todaytempview_margin_left">80.0px</dimen>
+    <dimen name="todaytempview_margin_top">75.0px</dimen>
     <dimen name="temparature_gap_s">2.0px</dimen>
-    <dimen name="temparature_gap_m">3.0px</dimen>
-    <dimen name="temparature_margin_top">-9.0px</dimen>
-    <dimen name="high_temp_margin_top">-1.0px</dimen>
-    <dimen name="low_temp_margin_top">-6.0px</dimen>
-    <dimen name="small_temp_width">30.0px</dimen>
+    <dimen name="temparature_gap_m">2.0px</dimen>
+    <dimen name="temparature_margin_top">-5.0px</dimen>
+    <dimen name="high_temp_margin_top">-8.0px</dimen>
+    <dimen name="low_temp_margin_top">-5.0px</dimen>
+    <dimen name="small_temp_width">22.0px</dimen>
     <dimen name="week_day_layout_width">108.0px</dimen>
     <dimen name="week_day_layout_height">20.0px</dimen>
     <dimen name="week_day_margin_top">0.0px</dimen>
-    <dimen name="week_day_margin_right">8.0px</dimen>
-    <dimen name="weatherclock_base_width">304.0px</dimen>
-    <dimen name="weatherclock_base_height">197.0px</dimen>
-    <dimen name="weatherclock_base_margin_left">8.0px</dimen>
-    <dimen name="weatherclock_base_margin_top">22.0px</dimen>
+    <dimen name="week_day_margin_right">4.0px</dimen>
+    <dimen name="weatherclock_base_width">229.0px</dimen>
+    <dimen name="weatherclock_base_height">114.0px</dimen>
+    <dimen name="weatherclock_base_margin_left">6.0px</dimen>
+    <dimen name="weatherclock_base_margin_top">12.0px</dimen>
     <dimen name="weather_view_margin_left">1.0px</dimen>
-    <dimen name="weather_view_margin_bottom">2.0px</dimen>
-    <dimen name="weather_press_width">300.0px</dimen>
-    <dimen name="weather_press_height">170.0px</dimen>
-    <dimen name="weather_press_margin_left">10.0px</dimen>
-    <dimen name="weather_press_margin_top">22.0px</dimen>
-    <dimen name="sun_width">110.0px</dimen>
-    <dimen name="sun_height">108.0px</dimen>
-    <dimen name="sun_margin_left">105.0px</dimen>
-    <dimen name="sun_margin_top">92.0px</dimen>
-    <dimen name="digitalclock_width">304.0px</dimen>
-    <dimen name="digitalclock_height">142.0px</dimen>
-    <dimen name="digitalclock_margin_top">-23.0px</dimen>
-    <dimen name="tab_width">149.0px</dimen>
-    <dimen name="tab_height">142.0px</dimen>
-    <dimen name="tab_margin_leftright">8.0px</dimen>
-    <dimen name="tab_down_height">60.0px</dimen>
+    <dimen name="weather_view_margin_bottom">5.0px</dimen>
+    <dimen name="weather_press_width">229.0px</dimen>
+    <dimen name="weather_press_height">114.0px</dimen>
+    <dimen name="weather_press_margin_left">6.0px</dimen>
+    <dimen name="weather_press_margin_top">12.0px</dimen>
+    <dimen name="sun_width">70.0px</dimen>
+    <dimen name="sun_height">70.0px</dimen>
+    <dimen name="sun_margin_left">85.0px</dimen>
+    <dimen name="sun_margin_top">87.0px</dimen>
+    <dimen name="digitalclock_width">229.0px</dimen>
+    <dimen name="digitalclock_height">87.0px</dimen>
+    <dimen name="digitalclock_margin_top">-12.0px</dimen>
+    <dimen name="tab_width">80.0px</dimen>
+    <dimen name="tab_height">87.0px</dimen>
+    <dimen name="tab_margin_leftright">26.0px</dimen>
+    <dimen name="tab_down_height">37.0px</dimen>
     <dimen name="tab_down_margin_top">0.0px</dimen>
-    <dimen name="tab_updown_margin_left">19.0px</dimen>
+    <dimen name="tab_updown_margin_left">0.0px</dimen>
     <dimen name="tab_up_margin_top">10.0px</dimen>
-    <dimen name="digital_width">48.0px</dimen>
-    <dimen name="digital_height">83.0px</dimen>
-    <dimen name="digital_up_height">40.0px</dimen>
-    <dimen name="digital_margin_top">24.0px</dimen>
-    <dimen name="digital_up_margin_top">24.0px</dimen>
-    <dimen name="digital_down_margin_top">-39.0px</dimen>
-    <dimen name="digital_ten_margin_left">25.0px</dimen>
-    <dimen name="digital_units_margin_left">77.0px</dimen>
-    <dimen name="digital_center_margin_left">51.0px</dimen>
-    <dimen name="container_down_width">149.0px</dimen>
-    <dimen name="container_down_height">60.0px</dimen>
-    <dimen name="container_down_margin_top">62.0px</dimen>
-    <dimen name="container_up_width">149.0px</dimen>
-    <dimen name="container_up_height">64.0px</dimen>
-    <dimen name="ampm_width">16.0px</dimen>
-    <dimen name="ampm_height">10.0px</dimen>
-    <dimen name="ampm_margin_left">26.0px</dimen>
-    <dimen name="ampm_margin_top">97.0px</dimen>
-    <dimen name="ampm_down_margin_top">34.0px</dimen>
-    <dimen name="mask_width">117.0px</dimen>
-    <dimen name="mask_height">56.0px</dimen>
-    <dimen name="mask_margin_left">17.0px</dimen>
-    <dimen name="mask_margin_top">63.0px</dimen>
-    <dimen name="divider_width">111.0px</dimen>
+    <dimen name="digital_width">34.0px</dimen>
+    <dimen name="digital_height">60.0px</dimen>
+    <dimen name="digital_up_height">34.0px</dimen>
+    <dimen name="digital_margin_top">14.0px</dimen>
+    <dimen name="digital_up_margin_top">14.0px</dimen>
+    <dimen name="digital_down_margin_top">-30.0px</dimen>
+    <dimen name="digital_ten_margin_left">4.0px</dimen>
+    <dimen name="digital_units_margin_left">42.0px</dimen>
+    <dimen name="digital_center_margin_left">23.0px</dimen>
+    <dimen name="container_down_width">80.0px</dimen>
+    <dimen name="container_down_height">37.0px</dimen>
+    <dimen name="container_down_margin_top">43.0px</dimen>
+    <dimen name="container_up_width">80.0px</dimen>
+    <dimen name="container_up_height">43.0px</dimen>
+    <dimen name="ampm_width">12.0px</dimen>
+    <dimen name="ampm_height">7.0px</dimen>
+    <dimen name="ampm_margin_left">2.0px</dimen>
+    <dimen name="ampm_margin_top">64.0px</dimen>
+    <dimen name="ampm_down_margin_top">21.0px</dimen>
+    <dimen name="mask_width">80.0px</dimen>
+    <dimen name="mask_height">37.0px</dimen>
+    <dimen name="mask_margin_left">0.0px</dimen>
+    <dimen name="mask_margin_top">43.0px</dimen>
+    <dimen name="divider_width">80.0px</dimen>
     <dimen name="divider_height">1.0px</dimen>
-    <dimen name="divider_margin_left">20.0px</dimen>
-    <dimen name="divider_margin_top">63.0px</dimen>
-    <dimen name="dualclock_citytext_size">13.0sp</dimen>
-    <dimen name="dualclock_citytext_height">20.0px</dimen>
-    <dimen name="dualclock_citytext_padding_left">9.0px</dimen>
-    <dimen name="dualclock_citytext_padding_right">9.0px</dimen>
-    <dimen name="dualclock_citytext_margin_bottom">8.0px</dimen>
-    <dimen name="dualclock_base_width">316.0px</dimen>
-    <dimen name="dualclock_base_height">146.0px</dimen>
-    <dimen name="dualclock_base_margin_top">18.0px</dimen>
-    <dimen name="dualclock_clocklayout_width">158.0px</dimen>
-    <dimen name="dualclock_clocklayout_height">175.0px</dimen>
+    <dimen name="divider_margin_left">0.0px</dimen>
+    <dimen name="divider_margin_top">42.0px</dimen>
+    <dimen name="dualclock_citytext_size">11.0px</dimen>
+    <dimen name="dualclock_citytext_height">16.0px</dimen>
+    <dimen name="dualclock_citytext_padding_left">5.0px</dimen>
+    <dimen name="dualclock_citytext_padding_right">5.0px</dimen>
+    <dimen name="dualclock_citytext_margin_bottom">3.0px</dimen>
+    <dimen name="dualclock_base_width">210.0px</dimen>
+    <dimen name="dualclock_base_height">90.0px</dimen>
+    <dimen name="dualclock_base_margin_top">8.0px</dimen>
+    <dimen name="dualclock_clocklayout_width">120.0px</dimen>
+    <dimen name="dualclock_clocklayout_height">128.0px</dimen>
     <dimen name="dualclock_clocklayout_margin_left">0.0px</dimen>
-    <dimen name="dualclock_clock_width">138.0px</dimen>
-    <dimen name="dualclock_clock_height">138.0px</dimen>
-    <dimen name="dualclock_clock_margin_top">11.0px</dimen>
-    <dimen name="dualclock_clock_ac_radius">34.0px</dimen>
-    <dimen name="dualclock_clock_ac_text_size">9.0px</dimen>
-    <dimen name="dualclock_clock_ac_data_width">25.0px</dimen>
-    <dimen name="dualclock_label_marginTop">157.0px</dimen>
+    <dimen name="dualclock_clock_width">104.0px</dimen>
+    <dimen name="dualclock_clock_height">104.0px</dimen>
+    <dimen name="dualclock_clock_margin_top">2.0px</dimen>
+    <dimen name="dualclock_clock_ac_radius">25.0px</dimen>
+    <dimen name="dualclock_clock_ac_text_size">7.0px</dimen>
+    <dimen name="dualclock_clock_ac_data_width">20.0px</dimen>
+    <dimen name="dualclock_label_marginTop">108.0px</dimen>
     <dimen name="dualclock_label_textsize">11.0px</dimen>
     <dimen name="dualclock_marginBottom">2.0px</dimen>
-    <dimen name="analogclock_clocklayout_width">160.0px</dimen>
-    <dimen name="analogclock_clocklayout_height">192.0px</dimen>
-    <dimen name="analogclock_clocklayout_padding_top">13.0px</dimen>
-    <dimen name="analogclock_citytext_size">13.0sp</dimen>
-    <dimen name="analogclock_citytext_height">20.0px</dimen>
-    <dimen name="analogclock_citytext_padding_left">9.0px</dimen>
-    <dimen name="analogclock_citytext_padding_right">9.0px</dimen>
-    <dimen name="analogclock_citytext_margin_bottom">0.0px</dimen>
-    <dimen name="analogclock_clock_width">146.0px</dimen>
-    <dimen name="analogclock_clock_height">146.0px</dimen>
-    <dimen name="analogclock_clock_margin_top">0.0px</dimen>
-    <dimen name="analogclock_clock_ac_radius">49.0px</dimen>
-    <dimen name="analogclock_clock_ac_text_size">9.0px</dimen>
-    <dimen name="analogclock_clock_ac_data_width">25.0px</dimen>
-    <dimen name="analogclock44_clocklayout_width">316.0px</dimen>
-    <dimen name="analogclock44_clocklayout_height">292.0px</dimen>
-    <dimen name="analogclock44_clocklayout_padding_top">13.0px</dimen>
-    <dimen name="analogclock44_citytext_size">13.0sp</dimen>
-    <dimen name="analogclock44_citytext_height">20.0px</dimen>
-    <dimen name="analogclock44_citytext_padding_left">9.0px</dimen>
-    <dimen name="analogclock44_citytext_padding_right">9.0px</dimen>
-    <dimen name="analogclock44_citytext_margin_bottom">0.0px</dimen>
-    <dimen name="analogclock44_clock_width">250.0px</dimen>
-    <dimen name="analogclock44_clock_height">250.0px</dimen>
+    <dimen name="analogclock_clocklayout_width">120.0px</dimen>
+    <dimen name="analogclock_clocklayout_height">128.0px</dimen>
+    <dimen name="analogclock_clocklayout_padding_top">2.0px</dimen>
+    <dimen name="analogclock_citytext_size">11.0px</dimen>
+    <dimen name="analogclock_citytext_height">16.0px</dimen>
+    <dimen name="analogclock_citytext_padding_left">5.0px</dimen>
+    <dimen name="analogclock_citytext_padding_right">5.0px</dimen>
+    <dimen name="analogclock_citytext_margin_bottom">3.0px</dimen>
+    <dimen name="analogclock_clock_width">104.0px</dimen>
+    <dimen name="analogclock_clock_height">104.0px</dimen>
+    <dimen name="analogclock_clock_margin_top">5.0px</dimen>
+    <dimen name="analogclock_clock_ac_radius">36.0px</dimen>
+    <dimen name="analogclock_clock_ac_text_size">7.0px</dimen>
+    <dimen name="analogclock_clock_ac_data_width">20.0px</dimen>
+    <dimen name="analogclock44_clocklayout_width">240.0px</dimen>
+    <dimen name="analogclock44_clocklayout_height">190.0px</dimen>
+    <dimen name="analogclock44_clocklayout_padding_top">1.0px</dimen>
+    <dimen name="analogclock44_citytext_size">11.0px</dimen>
+    <dimen name="analogclock44_citytext_height">16.0px</dimen>
+    <dimen name="analogclock44_citytext_padding_left">5.0px</dimen>
+    <dimen name="analogclock44_citytext_padding_right">5.0px</dimen>
+    <dimen name="analogclock44_citytext_margin_bottom">1.0px</dimen>
+    <dimen name="analogclock44_clock_width">169.0px</dimen>
+    <dimen name="analogclock44_clock_height">169.0px</dimen>
     <dimen name="analogclock44_clock_margin_top">0.0px</dimen>
-    <dimen name="analogclock44_clock_ac_radius">68.0px</dimen>
-    <dimen name="analogclock44_clock_ac_text_size">13.0px</dimen>
-    <dimen name="analogclock44_clock_ac_data_width">36.0px</dimen>
+    <dimen name="analogclock44_clock_ac_radius">48.0px</dimen>
+    <dimen name="analogclock44_clock_ac_text_size">10.0px</dimen>
+    <dimen name="analogclock44_clock_ac_data_width">30.0px</dimen>
     <dimen name="analogclock44_hit_width">150.0px</dimen>
     <dimen name="analogclock44_hit_height">250.0px</dimen>
     <dimen name="analogclock44_hit_margin_top">0.0px</dimen>
     <dimen name="analogclock44_hit2_width">250.0px</dimen>
     <dimen name="analogclock44_hit2_height">150.0px</dimen>
     <dimen name="analogclock44_hit2_margin_top">50.0px</dimen>
-    <dimen name="analogclockadj_clocklayout_width">316.0px</dimen>
-    <dimen name="analogclockadj_clocklayout_height">288.0px</dimen>
-    <dimen name="analogclockadj_clocklayout_padding_top">13.0px</dimen>
-    <dimen name="analogclockadj_citytext_size">13.0sp</dimen>
-    <dimen name="analogclockadj_citytext_height">20.0px</dimen>
-    <dimen name="analogclockadj_citytext_padding_left">9.0px</dimen>
-    <dimen name="analogclockadj_citytext_padding_right">9.0px</dimen>
-    <dimen name="analogclockadj_citytext_margin_bottom">0.0px</dimen>
+    <dimen name="analogclockadj_clocklayout_width">240.0px</dimen>
+    <dimen name="analogclockadj_clocklayout_height">190.0px</dimen>
+    <dimen name="analogclockadj_clocklayout_padding_top">0.0px</dimen>
+    <dimen name="analogclockadj_citytext_size">11.0px</dimen>
+    <dimen name="analogclockadj_citytext_height">16.0px</dimen>
+    <dimen name="analogclockadj_citytext_padding_left">5.0px</dimen>
+    <dimen name="analogclockadj_citytext_padding_right">5.0px</dimen>
+    <dimen name="analogclockadj_citytext_margin_bottom">1.0px</dimen>
     <dimen name="analogclockadj_hit_width">150.0px</dimen>
     <dimen name="analogclockadj_hit_height">250.0px</dimen>
     <dimen name="analogclockadj_hit_margin_top">0.0px</dimen>
     <dimen name="analogclockadj_hit2_width">250.0px</dimen>
     <dimen name="analogclockadj_hit2_height">150.0px</dimen>
     <dimen name="analogclockadj_hit2_margin_top">50.0px</dimen>
-    <dimen name="digitalclock_layout_padding_top">14.0px</dimen>
-    <dimen name="digitalclock_main_width">140.0px</dimen>
-    <dimen name="digitalclock_main_height">79.0px</dimen>
-    <dimen name="digitalclock_digital_width">30.0px</dimen>
-    <dimen name="digitalclock_digital_height">42.0px</dimen>
-    <dimen name="digitalclock_hour_tens_margin_left">3.0px</dimen>
-    <dimen name="digitalclock_hour_tens_margin_top">4.0px</dimen>
+    <dimen name="digitalclock_layout_padding_top">5.0px</dimen>
+    <dimen name="digitalclock_main_width">108.0px</dimen>
+    <dimen name="digitalclock_main_height">56.0px</dimen>
+    <dimen name="digitalclock_digital_width">20.0px</dimen>
+    <dimen name="digitalclock_digital_height">27.0px</dimen>
+    <dimen name="digitalclock_hour_tens_margin_left">9.0px</dimen>
+    <dimen name="digitalclock_hour_tens_margin_top">3.0px</dimen>
     <dimen name="digitalclock_hour_units_margin_left">-1.0px</dimen>
     <dimen name="digitalclock_dot_width">0.0px</dimen>
     <dimen name="digitalclock_dot_height">0.0px</dimen>
     <dimen name="digitalclock_dot_margin_left">0.0px</dimen>
     <dimen name="digitalclock_dot_margin_top">0.0px</dimen>
-    <dimen name="digitalclock_minute_tens_margin_left">15.0px</dimen>
+    <dimen name="digitalclock_minute_tens_margin_left">9.0px</dimen>
     <dimen name="digitalclock_minute_units_margin_left">-1.0px</dimen>
-    <dimen name="digitalclock_data_margin_left">8.0px</dimen>
-    <dimen name="digitalclock_data_margin_top">0.0px</dimen>
-    <dimen name="digitalclock_data_text_size">11.0sp</dimen>
-    <dimen name="digitalclock_ampm_width">17.0px</dimen>
-    <dimen name="digitalclock_ampm_height">10.0px</dimen>
-    <dimen name="digitalclock_ampm_margin_right">8.0px</dimen>
+    <dimen name="digitalclock_data_margin_left">7.0px</dimen>
+    <dimen name="digitalclock_data_margin_top">-2.0px</dimen>
+    <dimen name="digitalclock_data_text_size">12.0sp</dimen>
+    <dimen name="digitalclock_ampm_width">14.0px</dimen>
+    <dimen name="digitalclock_ampm_height">8.0px</dimen>
+    <dimen name="digitalclock_ampm_margin_right">5.0px</dimen>
     <dimen name="digitalclock_ampm_margin_top">3.0px</dimen>
-    <dimen name="digitalclock_divider_width">139.0px</dimen>
+    <dimen name="digitalclock_divider_width">103.0px</dimen>
     <dimen name="digitalclock_divider_height">2.0px</dimen>
-    <dimen name="digitalclock_divider_margin_left">0.0px</dimen>
-    <dimen name="digitalclock_divider_margin_top">2.0px</dimen>
-    <dimen name="digitalclock_city_width">129.0px</dimen>
-    <dimen name="digitalclock_city_margin_top">-1.0px</dimen>
-    <dimen name="digitalclock_city_text_size">12.0sp</dimen>
+    <dimen name="digitalclock_divider_margin_left">3.0px</dimen>
+    <dimen name="digitalclock_divider_margin_top">0.0px</dimen>
+    <dimen name="digitalclock_city_width">99.0px</dimen>
+    <dimen name="digitalclock_city_margin_top">-4.0px</dimen>
+    <dimen name="digitalclock_city_text_size">11.0px</dimen>
 </resources>
